apps/passport-client/src/dispatch.ts

```diff
@@ -166,9 +170,11 @@ export type Action =
   | {
       type: "handle-agreed-privacy-notice";
       version: number;
+      newUi?: boolean;
```

Comment:
let's make sure to not forget to remove this once phase 3 is completed. Let's track all such eventual cleanups here: https://github.com/proofcarryingdata/zupass/pull/1897/files#diff-854da518e499d3f8324df8073c58de1f74e1e243ec1ad68bd87015aa290d859cR173